PWGEM/PhotonMeson/TableProducer/createEMReducedEvent.cxx

```diff
@@ -42,6 +42,8 @@ struct CreateEMEvent {
   Produces<o2::aod::EMReducedEventsMult> event_mult;
   Produces<o2::aod::EMReducedEventsCent> event_cent;
 
+  Configurable<bool> requireCaloReadout{"requireCaloReadout", true, "Require calorimeters readout when analyzing EMCal/PHOS"};
```

Comment:
This is not what one should do. readout flag is stored in CCDB. This is handled by Evgeny Kryshen. If this CCDB is missing, can you ask him?

Reply:
You are right, this should not be necessary. However, the kTVXinEMC alias bit is unfortunately not correctly set in multiple MCs, so we would like to implement this bypass to be able to investigate the full statistics